plugin.py

```diff
@@ -41,9 +46,12 @@ def get_plugin_setting(cls, key: str, default: Optional[Any] = None) -> Any:
         return sublime.load_settings(cls.package_name + ".sublime-settings").get(key, default)
 
     @staticmethod
-    def find_package_dependency_dirs() -> List[str]:
+    def find_package_dependency_dirs(py_ver: str = "33") -> List[str]:
         dep_dirs = sys.path.copy()
 
+        if py_ver == "38":
+            dep_dirs = [re.sub(r"(python3\.?)3", r"\g<1>8", d, flags=re.IGNORECASE) for d in dep_dirs]
```

Comment:
Will this package eventually run in 3.8 host? If the plan is to make LSP run in 3.8 host (which I believe it is) then this will also have to run in 3.8. In which case maybe this logic should be more flexible to also account from switching from 3.8 to 3.3?

Reply:
done in f750f8a